src/cloud/components/Application.tsx

```diff
@@ -141,7 +140,9 @@ const Application = ({
 
   useEffect(() => {
     const handler = () => {
-      setShowFuzzyNavigation((prev) => !prev)
+      if (usingElectron) {
```

Comment:
Is there any reason why we blocking this behavior for the browser app?

Reply:
Hmm, it had some reason, it was either that it is covered somewhere else, so as not to duplicate it, 
or we had it in spec, or I thought it was something else and that web app does not have this search!
(looked at spec, seems it should be BOTH)